### PR TITLE
Max pages number

### PR DESCRIPTION
**Pull request by StayerX**
Now we can show the maximum number of pages

### CODE REVIEW

File: lib/Renard/Curie/Component/PageDrawingArea.pm [hunk 1]
```diff
@@ -102,7 +106,11 @@ sub setup_drawing_area {
 sub _trigger_current_page_number {
 	my ($self) = @_;
 	$self->refresh_drawing_area;
+}
 
+sub _trigger_max_pages_number {
```
Comment (zmughal) [inline lib/Renard/Curie/Component/PageDrawingArea.pm]: This function is not needed (not called).

File: lib/Renard/Curie/curie.glade [hunk 2]
```diff
@@ -247,6 +247,18 @@
               </packing>
             </child>
             <child>
+              <object class="GtkLabel" id="page-max-pages-entry">
```
Comment (zmughal) [inline lib/Renard/Curie/curie.glade]: I would name this to `number-of-pages-label`.

File: lib/Renard/Curie/Component/PageDrawingArea.pm [hunk 3]
```diff
@@ -68,6 +71,7 @@ sub on_draw_pdf_page {
 
 	$self->builder->get_object('page-number-entry')
 		->set_text($self->current_page_number);
+
```
Comment (zmughal) [inline lib/Renard/Curie/Component/PageDrawingArea.pm]: This extra whitespace can be removed.

File: lib/Renard/Curie/curie.glade (hunk 2 above)
Comment (zmughal) [inline lib/Renard/Curie/curie.glade]: @StayerX, what do you thing of renaming:

```
s/page-max-pages-entry/number-of-pages-label/  # Glade object ID
s/setup_max_page_number/setup_number_of_pages_label/  # function name
```

File: lib/Renard/Curie/Component/PageDrawingArea.pm [hunk 4]
```diff
@@ -178,4 +178,4 @@ sub toggle_backward_buttons {
 	}
 }
 
--1;
+1;
```
Comment (zmughal) [inline lib/Renard/Curie/Component/PageDrawingArea.pm]: We should squash this commit with the previous one.

File: lib/Renard/Curie/curie.glade (hunk 2 above)
Comment (StayerX) [inline lib/Renard/Curie/curie.glade]: ok

File: lib/Renard/Curie/Component/PageDrawingArea.pm (hunk 1 above)
Comment (StayerX) [inline lib/Renard/Curie/Component/PageDrawingArea.pm]: done

**Comment by StayerX** [comment]
You can review it again now. (this is for https://github.com/project-renard/curie/issues/21)

**Comment by zmughal** [comment]
Looks good to me! @ChiragGhanshani ?

**Comment by zmughal** [comment]
@StayerX, could you add an issue to test this when #41 is merged?

**Comment by zmughal** [comment]
Also, it would be good to attach a screenshot.

**Comment by StayerX** [comment]
curie-max-page-number

**Comment by zmughal** [comment]
This needs to be rebased before it can be merged. I'll take a look.

**Comment by zmughal** [comment]
I had to rebase the commit so that it is mergeable with the changes from #39.

I also fixed some whitespace problems (in the future, I will add an automatic whitespace checker).

**Comment by zmughal** [comment]
I think this is now ready for the tests as stated in issue #45.

**Comment by StayerX** [comment]
I think for the white spaces we can use lint or lint checker. I am not sure.

**Comment by zmughal** [comment]
Right, I've got an idea on how to do that using [`Perl::Critic`](http://p3rl.org/Perl::Critic).

**Comment by ChiragGhanshani** [comment]
Looks good to me.

**Comment by zmughal** [comment]
@StayerX, do you want to add tests for this before merging? It's up to you.

**Comment by StayerX** [comment]
Lets keep the good practice and not merge before the tests are done.

**Comment by zmughal** [comment]
Running:

```
git fetch origin
git rebase origin/master  # get all the latest changes from master
git push -f origin max-pages-number
```

**Comment by zmughal** [comment]
@StayerX, go ahead and merge! The tests pass!

**Pull request merged**